test/app-luatest/console_debugger_session_test.lua

```diff
@@ -0,0 +1,56 @@
+local t = require('luatest')
+local popen = require('popen')
+local g = t.group()
+
+local function normalize_path(s)
+    return s:gsub("^@", ""):gsub("[^/]+$", "")
+end
+
+local function unescape(s)
+    return s:gsub('[\27\155][][()#;?%d]*[A-PRZcf-ntqry=><~]', '')
+end
+
+local TARANTOOL_PATH = arg[-1]
+local path_to_script = normalize_path(debug.getinfo(1,'S').source)
+local debug_target_script = path_to_script .. 'debug-target.lua'
+
+local DEBUGGER = 'cdebugger.lua'
+local dbg_header = DEBUGGER .. ": Loaded for ".. jit.version
+local dbg_prompt = DEBUGGER..'>'
+
+local sequence = {
+    {[''] = dbg_header},
```

Comment:
I've created simple luatest script for injecting `n` and `p` commands into interactive debugger shell. At the moment it assumes that debugger got invoked from inside of a Lua script itself (unless we have externals means to activated debugger, which is the next step)